gui/src/components/Sidebar.svelte

```diff
@@ -27,11 +29,28 @@
   }
 
   $: loading = !$versionStore;
+  $: {
+    switch ($currentGame) {
+      case "jumpy":
+        gameTitle = "Jumpy";
+        break;
+      case "punchy":
+        gameTitle = "Punchy";
+        break;
+    }
+  }
```

Comment:
it would be nice if we create Game object as contant

```ts
// constants.ts
//...
export const GAMES = {
    JUMPY: 'Jumpy',
    PUNCHY: 'Punchy'
} 
```

and we can use `GAMES` key as type

```ts
// global.d.ts

export type Game = keyof typeof GAMES; // "JUMPY" | "PUNCHY"

```

So we can write in a simpler way

```suggestion
  $: {
       gameTitle = GAMES[$currentGame]
  }
```